tests/functional/a/assert_on_tuple.txt

```diff
@@ -1,6 +1,6 @@
-assert-on-tuple:4:0:4:35::Assert called on a populated tuple. Did you mean 'assert x,y'?:UNDEFINED
+assert-on-tuple:4:0:4:81::Assert called on a populated tuple. Did you mean 'assert x,y'?:UNDEFINED
```

Comment:
This should have HIGH at the end. You can update the test automatically (a command is given when a functional test fail)

Reply:
AssertionError: Wrong output for 'assert_on_tuple.txt':
E       You can update the expected output automatically with: 'python tests/test_functional.py --update-functional-output -k "test_functional[assert_on_tuple]"'